src/covid_model_seiir_pipeline/lib/ihme_deps.py

```diff
@@ -0,0 +1,35 @@
+"""Wrappers for IHME specific dependencies.
+
+This module explicitly declares and wraps IHME specific code bases
+to prevent CI failures at import time.
+"""
+
+
+def _lazy_import_error_object(module_name: str):
+
+    class C:
+        def __init__(self, *args, **kwargs):
+            raise ModuleNotFoundError(f"No module named '{module_name}'")
+
+
+def _lazy_import_error_function(module_name: str):
+    def f(*args, **kwargs):
+        raise ModuleNotFoundError(f"No module named '{module_name}'")
+
+
+try:
+    from db_queries import get_location_metadata
+except ModuleNotFoundError:
+    get_location_metadata = _lazy_import_error_function('db_queries')
+
+try:
+    from jobmon.client import Workflow, BashTask
+    from jobmon.client.swarm.executors.base import ExecutorParameters
+    from jobmon.client.swarm.workflow.task_dag import DagExecutionStatus
+    from jobmon.client.swarm.workflow.workflow import WorkflowAlreadyComplete
+except ModuleNotFoundError:
```

Comment:
The way you've written this you have to update the list of imports in the `try` and `except` any time there is change.

Suggest re-writing this so that you have a single list of imports. This could be as easy as updating `_lazy_import_foo` to do this

Reply:
Updated and with tests even (though only happy path, really)!